AgriParcel/schema.json

```diff
@@ -1,162 +1,180 @@
+                    }
+                ],
+                "description": "Relationship. Reference to the crop associated with this parcel"
+            },
+            "hasAgriSoil": {
+                "anyOf": [
+                    {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 256,
+                        "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
+                        "description": "Property. Identifier format of any NGSI entity"
+                    },
+                    {
+                        "type": "string",
+                        "format": "uri",
+                        "description": "Property. Identifier format of any NGSI entity"
+                    }
+                ],
+                "description": "Relationship. Reference to the soil associated with this parcel of land"
+            },
+            "cropStatus": {
+                "type": "string",
+                "description": "Property. Enum:'seeded, justBorn, growing, maturing, readyForHarvesting'. A choice from an enumerated list describing the crop planting status",
+                "enum": [
+                    "seeded",
+                    "justBorn",
+                    "growing",
+                    "maturing",
+                    "readyForHarvesting"
+                ]
+            },
+            "lastPlantedAt": {
+                "type": "string",
+                "format": "date-time",
+                "description": "Property. Indicates the date when the crop was last planted"
+            },
+            "hasDevices": {
+                "type": "array",
+                "description": "Property. Reference to the IoT devices associated with this parcel i.e. sensors, controls.",
+                "items": {"$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/EntityIdentifierType"}
+            },
+            "soilTextureType": {
+                "type": "string",
+                "description": "Property. Enum: 'Clay', 'Clay, Sandy', 'Clay, Silty', 'Loam', 'Loam, Clay', 'Loam, Sandy', 'Loam, Sandy Clay', 'Loam, Silty', 'Loam, Silty Clay', 'Sand', 'Sand, Loamy', 'Silt'",
+                "enum": [
+                    "Clay",
+                    "Clay, Sandy",
+                    "Clay, Silty",
+                    "Loam",
+                    "Loam, Clay",
+                    "Loam, Sandy",
+                    "Loam, Sandy Clay",
+                    "Loam, Silty",
+                    "Loam, Silty Clay",
+                    "Sand",
+                    "Sand, Loamy",
+                    "Silt"
+                ]
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
+            "irrigationSystemType": {
```

Comment:
where do these categories come from? If any regulation links it/them in the description. Can be two of them at the same time?

Reply:
I´ve committed the changes we discussed in the latest version. Made some changes to enum values in soilTextureType and added references to where the enum values for soilTextureType and irrigationSystemType are defined.